include/gul14/num_util.h

```diff
@@ -71,16 +71,17 @@ constexpr auto abs(ValueT n) noexcept -> std::enable_if_t<not std::is_unsigned<V
  * orders equals digits does not hold so strict anymore.
  *
  * Remember that any nonzero number has infinite different significant digits compared
- * with 0.00000000. So if either a or b is 0.0 the result must be false.
+ * with 0.00000000. So if one operand is 0.0 while the other is not 0.0 the result must be false.
  *
  * \param a       The first number to compare
  * \param b       The second number to compare (same type as a)
  * \param orders  The number of digits to take for comparison (any numeric type)
  *
- * \returns true if the difference between a and b is orders of magnitude lower than the
- *          value of a or b.
+ * \returns true if a and b are equal or the difference between a and b is orders of
```

Comment:
Maybe we could also make the end of the sentence more explicit – I always fail to read it in the intended way: ... is \c orders orders of magnitude lower ... (or add quotation marks or whatever)

Reply:
Hmm, the `\c` does switch to a monospaced font (I guess), but on my machine the difference is almost invisible.
Would have liked a colored background or something. Maybe one can change that in the Doxy settings file?

![image](https://user-images.githubusercontent.com/16012374/195594082-52aa4e78-049a-405d-ab71-5f9ec781ee45.png)

Anyhow, changed.